### DISCUSSION TITLE
Clips on two streams

### DISCUSSION THREAD

**Discussion by ORi0N** [blakeblackshear/frigate#1138] [Ideas]
I have two camera's watching the same zone from a different angle. I'd like only one camera to DETECT the motion due to lots of bushes movement on the other one, but capture the clips on the TWO camera's when there is movement on the one that's positioned better. I tried adding the paths to both camera's under the same input and two times 'clips' but that's prohibited.

Could this be allowed please? I recon it's just adding multiple triggers on the same detection?

**Comment by blakeblackshear** [comment]
It's not quite that simple. I think the feature request to trigger events manually should convert this use case

**Reply by ORi0N** [reply]
It could indeed, by listening to motion event on the first camera and triggering a clip recording on the second one through Node-Red or HA automation.

But if multiple 'clips' on the input paths would be allowed, this could prove to be a nice feature as well, no? I mean, you could have a full overview of the house once motion was detected in a specific area?

Anyway, just thinking out loud here :)
